### ISSUE TITLE
Ads came back

### ISSUE THREAD

**Issue by Kuro-23** [Pyenb/Outlook-desktop-ad-blocker#2]
Today I got the ads even with 0.0.0.0 outlookads.live.com applied in the hosts file. Maybe microsoft changed the url?

**Comment by Pyenb** [comment]
Hm, could be. I will take a look at this when I have time.
I switched to Thunderbird in the meantime, but I still have outlook installed, so should be no problem 👍

**Comment by Kuro-23** [comment]
It's working great. thx!

**Comment by 7ergei** [comment]
> new ad url: https://m.adnxs.com/ut/v3 block this

Hi, can you please explain how to use this?

**Comment by Pyenb** [comment]
I updated the script. @ABucket Thank you for the input. Do you know if I'll still need to block both domains or only that one? I don't really have the time currently to check.

**Comment by 7ergei** [comment]
Installed new script and it works, thanks @Pyenb !

**Comment by ABucket** [comment]
> I updated the script. @ABucket Thank you for the input. Do you know if I'll still need to block both domains or only that one? I don't really have the time currently to check.

Only that one, there's another domain `https://www.bing.com/api/v1/mediation/clientauction` that also contains the ads, but seems like blocking that one is enough

**Comment by Pyenb** [comment]
Ah ok, thanks. As long as it's one the single one it's relatively easy. But with host files you can only block the domain as a whole, so to block `https://www.bing.com/api/v1/mediation/clientauction`, you would need to  block the entire `bing.com` domain. But as long as it's only the subdomain for now, it won't be a problem.

**Comment by Primexus** [comment]
Here's what it should look like if you wish to do it manually.
```
0.0.0.0 outlookads.live.com
0.0.0.0 m.adnxs.com
```

**Comment by Kuro-23** [comment]
I have those 2 in the host file:
0.0.0.0 outlookads.live.com
0.0.0.0 m.adnxs.com
but now I get microsoft outlook own ad... "Upgrade your account to premium" or something mine is in Italian.

**Comment by simpledimples** [comment]
I also have these 2 in the host file:
0.0.0.0 outlookads.live.com
0.0.0.0 m.adnxs.com

but getting "Upgrade your account" ads 
![image](https://github.com/user-attachments/assets/a5005b8e-8641-4258-924a-d6b9fcaaeacf)

EDIT: https://github.com/valinet/NewOutlookPatcher works, no ads and not even any placeholder blank space where the ads were - just straight to the emails

**Comment by Pyenb** [comment]
Yeah, it's a good repo. While being more intrusive, it really removes everything. I will still have a look if I can remove the new ads

**Comment by Kiwironic** [comment]
0.0.0.0 outlookads.live.com
0.0.0.0 m.adnxs.com
0.0.0.0 adnxs.com
0.0.0.0 acdn.adnxs.com
0.0.0.0 3lift.com

**Comment by Pyenb** [comment]
Thanks, will update the script shortly, unless someone wants to make a PR first. I am busy atm and sadly don't find a lot of time.

**Comment by L0que** [comment]
Do we have a solution to the 'Upgrade your account' ad that does not go away, at least for me, even with all of Kiwironic's longer list of urls?

**Comment by cavazos-apps** [comment]
+1 To still receiving ads to upgrade

**Comment by Kiwironic** [comment]
127.0.0.1 outlookads.live.com
127.0.0.1 m.adnxs.com
127.0.0.1 adnxs.com
127.0.0.1 www.adnxs.com
127.0.0.1 acdn.adnxs.com
127.0.0.1 3lift.com
127.0.0.1 www.3lift.com
127.0.0.1 browser.events.data.microsoft.com/OneCollector/
127.0.0.1 m.adnxs.com/ut/v3
127.0.0.1 eb2.3lift.com

**Comment by Kiwironic** [comment]
I can confirm the above works, but they have updated their JavaScript code so that when they fail to show ads, they will show this instead. 

![Image](https://github.com/user-attachments/assets/92b8ec80-3250-4912-a5a0-907ae9133440)

One way to get rid of that is to use this to inject the DLL. 
https://github.com/valinet/NewOutlookPatcher 

This works in removing the ads box itself. Instead of rewriting the url for ad servers, killing a process or running a scheduled task to remove the ads (which leaves the ads box empty but visible).

**Comment by Pyenb** [comment]
Sorry for the long wait. Thank you so much for your work @Kiwironic 

I will try and update the tool to include all of these.

**Comment by Girofox** [comment]
Does this method still work? 

In Process Monitor (with resolute network addresses on) **msedgewebview2.exe** shows only network requests to specific IP addresses and `954.bm-nginx-loadbalancer.mgmt.ams3.adnexus.net` for example.

Adnexus sounds like something ad related. 

When i block this address via host file the three digit number at the beginning changes and ads are still there. 😒

**Comment by Kiwironic** [comment]
I gave up on the cat and mouse, now I am using this method: 
https://github.com/valinet/NewOutlookPatcher

**Comment by tute123456** [comment]
This patcher works fine with W11 24H2 26120.3964 !!!!!!!
[https://github.com/valinet/NewOutlookPatcher](url)
Thx!

**Comment by MidnightRider73** [comment]
I'm still getting ads even with newoutlookpatcher installed.

**Comment by vasilli-ra** [comment]
Not working on Win11

**Comment by ThiWebCreative** [comment]
Not working anymore...

ThiWeb

**Comment by JuaKali219** [comment]
Tried all of the methods but none of them changed anything at all. Every reboot of outlook same old ads.
